src/guiguts/root.py

```diff
@@ -10,13 +11,22 @@
 
 from guiguts.maintext import maintext
 from guiguts.preferences import preferences, PrefKey
+from guiguts.utilities import is_x11
```

Comment:
Not on Linux, I'm afraid

Reply:
For extra fun, Linux is in the midst of a lengthy transition away from X11 toward Wayland. It may be X11 will still linger and be running for 20 more years, or we might, eventually, have to figure that out... but it's not for today, in any case!

Reply:
Great fun! 😁

Reply:
ATM, I don't think there's much/anything being done to make python/tkinter work natively (if that's the right word) with Wayland, though everyone seems to be saying there is so much old code relying on X11, that Wayland will have an integrated X server for a long time to come.